src/views/Home/Home.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import DogList from '../DogCard/DogList';
+import './Home.css';
+
+export default function Home() {
+  return (
+    <div>
+      <div className='newdog'>
+        <h3 >Have a friend looking for forever home?</h3> 
+        <Link exact to = '/dogs/new'>
+          <h3 className='newdoglink'> Click Here!</h3>
+        </Link>
```

Comment:
Here we will want to check for a currentUser and hide the new dog link and display a sign-in link. This can be done by using a ternary looking something like this => currentUser ? Click Here! : Sign-in